test/scrabble_test.rb

```diff
@@ -1,13 +1,45 @@
 gem 'minitest'
 require_relative '../lib/scrabble'
 require 'minitest/autorun'
-require 'minitest/emoji'
+require 'minitest/pride'
+#require 'minitest/emoji' *doesn't work*
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_two_letter_word
```

Comment:
Test suite looks nice! Breaking into smaller methods would allow you to add some unit tests.